src/main/java/com/google/plantasy/utils/GameUtils.java

```diff
@@ -31,7 +31,7 @@
   /**
   * Creates a game entity and returns the entity if successful
   */
-  public static Entity createGameEntity(String gameName, DatastoreService datastore) {
+  public static Game createGameEntity(String gameName, DatastoreService datastore) {
```

Comment:
This should be renamed to just createGame() considering we're no longer returning an entity. (I'm not sure if any other methods are now similarly misnamed but if any are, they should be fixed as well)